lib/auxiliary/sysinfo_parser.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'csv'
+require 'active_support'
+require 'active_support/core_ext/object/deep_dup'
```

Comment:
Sorry for repeatedly adding more comments, but I found more unused code here.

Reply:
we need a review to make the code cleaner, so not a problem

Reply:
removed